### PR TITLE
rafa-jaume-solution

### PR DESCRIPTION
**Pull request by jaumejane90**

### CODE REVIEW

File: src/board.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Cell from './cell'
+
+export default function Board(cellsMatrix) {
+	this.cellsMatrix = cellsMatrix
```
Comment (rafinskipg) [inline src/board.js]: please use 2 spaces indentation

File: src/board.spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const matrix = [[true, true , true], [true, true, true], [true, true, true]]
+
+describe('Board', () => {
+  it('should be created', () => {
+    var board = new Board(matrix)
+    board.initialize()
+    board.print()
+    expect(board).to.be.a('object')
+  })
+
```
Comment (rafinskipg) [inline src/board.spec.js]: Add more test cases, including print. 

You can actually pass to the board.print the method that outputs strings to the console if you want to mock it (you have to mock it) :+1:

File: src/cell.spec.js [hunk 3]
```diff
@@ -3,7 +3,21 @@ import Cell from './cell'
...
+    cell.toggle()
+    expect(cell.active).to.be.false
+  })
+
+  it('should toggle to true', () => {
+    var cell = new Cell(false)
+    cell.toggle()
+    expect(cell.active).to.be.true
+  })
+
```
Comment (rafinskipg) [inline src/cell.spec.js]: ensure the print method works correctly

File: src/game.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import Board from './board'
+
```
Comment (rafinskipg) [inline src/game.js:2]: Congratulations

File: src/game.spec.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import Game from './game'
+
+describe('Game', () => {
+  it('should be created', () => {
+    var game = new Game()
+    game.rerender()
```
Comment (rafinskipg) [inline src/game.spec.js]: you are not testing this function actually.

**Review by rafinskipg — CHANGES_REQUESTED** [review]
Some changes on testing and indentation

**Comment by jaumejane90** [comment]
@rafinskipg I think i fixed all your comments! Feel free to keep discussion up! cc @sloy

**Pull request closed without merging**